src/applications/mhv-landing-page/tests/containers/AppConfig.unit.spec.jsx

```diff
@@ -8,73 +8,54 @@ import { expect } from 'chai';
 import AppConfig from '../../containers/AppConfig';
 import { appName } from '../../manifest.json';
 
-const initialStateFn = ({ accountUuid = 'test-id' } = {}) => ({
-  user: {
-    profile: {
-      accountUuid,
-    },
-  },
-});
-
 describe(`${appName} -- <AppConfig />`, () => {
-  it('initializes datadog RUM', () => {
-    const useRumSpy = sinon.spy(datadogRum, 'init');
-
-    const initialState = initialStateFn();
-
-    const { getByText } = renderWithStoreAndRouter(
-      <AppConfig>
-        <p>child node</p>
-      </AppConfig>,
-      { initialState },
-    );
-
-    waitFor(() => {
-      getByText('child node');
-
-      // Check that datadogRum methods are called
-      expect(useRumSpy.called).to.be.true;
+  describe('calls datadogRum.setUser', () => {
+    const initialStateFn = ({ accountUuid = 'test-id' } = {}) => ({
+      user: {
+        profile: {
+          accountUuid,
+        },
+      },
     });
 
-    useRumSpy.restore();
-  });
-
-  it('calls datadogRum.setUser when a user id is available', () => {
-    const setRumUserSpy = sinon.spy(datadogRum, 'setUser');
-    const initialState = initialStateFn();
-
-    const { getByText } = renderWithStoreAndRouter(
-      <AppConfig>
-        <p>child node</p>
-      </AppConfig>,
-      { initialState },
-    );
+    let setRumUserSpy;
+    beforeEach(() => {
+      setRumUserSpy = sinon.spy(datadogRum, 'setUser');
```

Comment:
Note: This needs to be set and restored for each test. If the first test fails, it will stop and not restore the spy. Then, the second test would have failed because you cannot set a spy twice.